### PR TITLE
Added error printing

### PR DESCRIPTION
**Pull request by erocheleau**
on response code != 200 on analytics event

### CODE REVIEW

File: analytics/client.go [hunk 1]
```diff
@@ -271,6 +272,10 @@ func (c *client) sendRawEventRequest(method string, path string, body string) er
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != http.StatusOK {
+		fmt.Printf("\nRequest response error (%d): %s\n", resp.StatusCode, string(body))
```
Comment (pastjean) [inline analytics/client.go]: Replace fmt for log

**Review by pastjean — APPROVED** [review]

File: analytics/client.go (hunk 1 above)
Comment (pastjean) [inline analytics/client.go:276]: replace fmt for log

**Review by pastjean — CHANGES_REQUESTED** [review]

File: analytics/client.go (hunk 1 above)
Comment (erocheleau) [reply analytics/client.go:276]: You use log?

Comment (erocheleau) [reply analytics/client.go:276]: Will replace by just generating and returning an error with the body of the response and the response status.

**Pull request closed without merging**